packages/core/src/lib/core/file-storage/helpers/file-storage-factory.ts

```
@@ -0,0 +1,20 @@
+import multer from 'multer';
+import * as path from 'path';
+import { FileStorage } from '../file-storage';
+import { DirectoryPathGenerator } from './directory-path-generator';
+import { IDirectoryPathGenerator } from './directory-path-generator.interface';
+
+// FileStorageFactory
+export class FileStorageFactory {
+	private static readonly pathGenerator: IDirectoryPathGenerator = new DirectoryPathGenerator();
+
+	public static create(baseDirname: string): multer.StorageEngine {
+		const baseDirectory = this.pathGenerator.getBaseDirectory(baseDirname);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)